jvb/src/main/java/org/jitsi/videobridge/cc/BandwidthProbing.java

```diff
@@ -30,27 +30,27 @@
 /**
   * @author George Politis
   */
- public class BandwidthProbing
+ public abstract class BandwidthProbing
```

Comment:
`BandwidthProbing` isn't that big a class - would it be easier/cleaner just to convert the whole thing to Kotlin?

Reply:
Sure, I can.  I was just trying to keep things to the bare minimum needed for the `Bandwidth` class change.  Think I should do the rest as part of this? If not I can do it in the follow up (of which there will probably be a couple, I'm trying to keep diff sizes reasonable for PRs)

Reply:
For `BandwidthProbing` specifically, I think that moving it in its entirety isn't that much bigger than this change, because of all the `protected` changes and the like that the Java/Kotlin split implies.

Reply:
The remaining changes weren't trivial and I'm a bit nervous given how critical BandwidthProbing is, but I pushed the change and we can decide to keep it here or split it out.